### PR TITLE
The Feed Screen

### PR DESCRIPTION
**Pull request by tbezman**

### CODE REVIEW

**Comment by tbezman** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1399** <a href="https://app.graphite.dev/github/pr/gallery-so/gallery/1399" target="_blank">Graphite</a> 
    * **PR #1400** <a href="https://app.graphite.dev/github/pr/gallery-so/gallery/1400" target="_blank">Graphite</a> 
      * **PR #1401** <a href="https://app.graphite.dev/github/pr/gallery-so/gallery/1401" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/gallery-so/gallery/1401?utm_source=stack-comment).

**Comment by tbezman** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1399** <a href="https://app.graphite.dev/github/pr/gallery-so/gallery/1399" target="_blank">Graphite</a> 
    * **PR #1400** <a href="https://app.graphite.dev/github/pr/gallery-so/gallery/1400" target="_blank">Graphite</a> 
      * **PR #1401** <a href="https://app.graphite.dev/github/pr/gallery-so/gallery/1401" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/gallery-so/gallery/1401?utm_source=stack-comment).

File: .yarnrc.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 nodeLinker: node-modules
 
+checksumBehavior: update
```
Comment (tbezman) [inline .yarnrc.yml:3]: Been seeing some issues resolving merge conflicts with checksums. Turning this on fixes the problem so I figured I'd just leave it on.

File: apps/mobile/package.json [hunk 2]
```diff
@@ -23,6 +22,7 @@
     "react-native-fast-image": "^8.6.3",
     "react-native-fetch-api": "^3.0.0",
     "react-native-gesture-handler": "~2.9.0",
+    "react-native-markdown-display": "https://github.com/jonasmerlin/react-native-markdown-display.git",
```
Comment (tbezman) [inline apps/mobile/package.json:25]: forked version of react native markdown display

File: apps/mobile/src/components/Markdown.tsx
Comment (tbezman) [inline apps/mobile/src/components/Markdown.tsx:1]: I'm sure this file will evolve in the future

File: apps/mobile/src/components/Feed/FeedList.tsx [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+          markEventAsFailure(item.event.dbid);
+        }, []);
+
+        return null;
+      }
+
+      switch (item.kind) {
+        case 'header':
+          return (
+            <ReportingErrorBoundary fallback={<Fallback />}>
```
Comment (tbezman) [inline apps/mobile/src/components/Feed/FeedList.tsx:115]: Swap the fallback out with `onError`

File: apps/mobile/src/components/Feed/FeedListCollectorsNote.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+import { Markdown } from '../Markdown';
+
+type Props = {
+  collectorsNote: string;
+};
+
+export function FeedListCollectorsNote({ collectorsNote }: Props) {
+  return (
+    <View className="flex flex-row space-x-2 px-3 py-2">
+      <View className="bg-porcelain h-full w-[2]" />
```
Comment (tbezman) [inline apps/mobile/src/components/Feed/FeedListCollectorsNote.tsx:12]: Can use `.5` here for width

File: apps/mobile/src/components/Feed/GalleryUpdatedFeedEvent.tsx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  const renderItem = useCallback<ListRenderItem<(typeof subEvents)[number]>>(
+    ({ item }) => {
+      return <NonRecursiveFeedListItem eventCount={subEvents.length} eventDataRef={item} />;
+    },
+    [subEvents.length]
+  );
+
+  return (
+    <View className="flex flex-col space-y-3">
+      <FlatList
```
Comment (tbezman) [inline apps/mobile/src/components/Feed/GalleryUpdatedFeedEvent.tsx:68]: Leave comment about height issues w/ FlashList

File: apps/mobile/src/components/NftPreview.tsx [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+      fragment NftPreviewFragment on Token {
+        __typename
+        ...getVideoOrImageUrlForNftPreviewFragment
+      }
+    `,
+    tokenRef
+  );
+
+  const previews = getVideoOrImageUrlForNftPreview(token);
+  const tokenUrl = previews?.urls.medium;
```
Comment (tbezman) [inline apps/mobile/src/components/NftPreview.tsx:30]: why am i not handling video more explicitly?

File: apps/mobile/src/components/SvgWebView.tsx [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+        if (uri.match(/^data:image\/svg/)) {
+          const index = uri.indexOf('<svg');
+          setSvgContent(uri.slice(index));
+        } else {
+          try {
+            const res = await fetch(uri, { signal });
+            const text = await res.text();
+            setSvgContent(text);
+          } catch (err) {
+            console.error('got error', err);
```
Comment (tbezman) [inline apps/mobile/src/components/SvgWebView.tsx:63]: handle error better

**Review by Robinnnnn — APPROVED** [review]
🔥🔥🔥🔥🔥🔥🔥

**Pull request merged**